test/OptionSettlementEngine.t.sol

```diff
@@ -162,6 +162,17 @@ contract OptionSettlementTest is Test, NFTreceiver {
         assertTrue(!claim.claimed);
     }
 
+    function testWriteClaimWithOptionId() public {
```

Comment:
Based on current test suite layout, we should move this test into FAIL TESTS section and rename to `testRevertWriteClaimWithOptionId` or similar.